### ISSUE TITLE
Cannot import 'GradientDescentOptimizer'

### ISSUE THREAD

**Issue by scanfyu** [tensorflow/tensorflow#34151]
![image](https://user-images.githubusercontent.com/3198940/68577281-bb353800-04aa-11ea-8a50-0a48d2ad0974.png)

I use train.GradientDescentOptimizer and tf.optimizers.GradientDescentOptimizer, which not work. So I read the doc but only search GradientDescentOptimizer  in tf.compat.v1.train.GradientDescentOptimizer

**Comment by nikochiko** [comment]
See the documentation for [tf.train](https://www.tensorflow.org/api_docs/python/tf/train) and [tf.optimizers](https://www.tensorflow.org/api_docs/python/tf/optimizers). According to these, we do not have a batch gradient descent optimizer in either. There is Stochastic Gradient Descent optimizer in `tf.optimizers.SGD`. For gradient descent optimizer, so the way to apply Gradient Descent is `tf.compat.v1.train.GradientDescentOptimizer`.

**Comment by scanfyu** [comment]
It works for me.Thanks.
